src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,137 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int POSITION_FOR_NULL_KEYS = 0;
+    private int size;
+    private int mapCapacity;
```

Comment:
why do we need `mapCapacity` parameter?

Reply:
I use it in hash method and in resize method